podcasts/MiniPlayerViewController+Positioning.swift

```diff
@@ -37,11 +37,48 @@ extension MiniPlayerViewController {
         })
     }
 
+    func showMiniPlayerWithDelay() {
+        // only show if something is playing
+        if PlaybackManager.shared.currentEpisode() == nil { return }
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+            UIView.animate(withDuration: 0.2, animations: { () in
+                self.moveToShownPosition()
+            }, completion: { _ in
+                self.moveToShownPosition() // call this again in case the animation block wasn't called. It's ok to call this twice
+                NotificationCenter.postOnMainThread(notification: Constants.Notifications.miniPlayerDidAppear)
+            })
+        }
+    }
+
     func openFullScreenPlayer(completion: (() -> Void)? = nil) {
         guard PlaybackManager.shared.currentEpisode() != nil else { return }
 
         if playerOpenState == .open || playerOpenState == .animating { return }
 
+        guard !FeatureFlag.newPlayerTransition.enabled else {
+            guard let fullScreenPlayer else {
+                return
+            }
+
+            playerOpenState = .animating
+            aboutToDisplayFullScreenPlayer()
```

Comment:
When tapping the mini player the fullScreenPlayer check will always be nil since it's set in `aboutToDisplayFullScreenPlayer`. 

```suggestion
            aboutToDisplayFullScreenPlayer()

            guard let fullScreenPlayer else {
                return
            }

            playerOpenState = .animating

```